Analysis/Tasks/PWGHF/HFTrackIndexSkimsCreator.cxx

```diff
@@ -30,41 +31,77 @@ using namespace o2::framework;
 using namespace o2::framework::expressions;
 using namespace o2::aod::hf_cand_prong2;
 using namespace o2::aod::hf_cand_prong3;
+using namespace o2::analysis;
+using namespace o2::analysis::hf_cuts_single_track;
 
 /// Track selection
 struct SelectTracks {
+
+  // enum for candidate type
+  enum candidateType {
+    k2Prong = 0,
+    k3Prong
```

Comment:
Since `enum` values are supposed to start with an upper case letter, I suggest to change these to `Cand2Prong` and `Cand3Prong`.

Reply:
Done!